src/json-output.ts

```diff
@@ -58,12 +66,18 @@ export function jsonOutput(results: ResultStatsWithDifferences[]):
     }
     benchmarks.push({
       name: result.result.name,
-      samples: result.result.millis,
+      bytesSent: result.result.bytesSent,
+      version: result.result.version ? result.result.version : undefined,
+      browser: {
+        ...result.result.browser,
+        userAgent: result.result.userAgent,
+      },
       mean: {
         low: result.stats.meanCI.low,
         high: result.stats.meanCI.high,
       },
       differences,
+      samples: result.result.millis,
```

Comment:
I moved `samples` to the end since the order of fields here determines the order fields appear in the JSON file. Since this array is likely to be pretty long, I thought it made sense to put it last so it doesn't push the shorter fields at the top below the fold when first opening the file.